internal/provider/data_source_auth0_client_test.go

```diff
@@ -24,7 +25,7 @@ data auth0_client test {
 `
 
 func TestAccDataClientByName(t *testing.T) {
-	httpRecorder := configureHTTPRecorder(t)
+	httpRecorder := recorder.New(t)
```

Comment:
We are now wrapping the go-vcr recorder into our own type Recorder within it's own package, so it's easier to swap with something else or perform upgrades.